lib/iris/coords.py

```diff
@@ -2775,6 +2778,469 @@ def xml_element(self, doc):
         return cellMethod_xml_element
 
 
+class Connectivity(_DimensionalMetadata):
+    """
+    A CF-UGRID topology connectivity, describing the topological relationship
+    between two lists of dimensional locations. One or more connectivities
+    make up a CF-UGRID topology - a constituent of a CF-UGRID mesh.
+
+    See: https://ugrid-conventions.github.io/ugrid-conventions
+
+    """
+
+    UGRID_CF_ROLES = [
+        "edge_node_connectivity",
+        "face_node_connectivity",
+        "face_edge_connectivity",
+        "face_face_connectivity",
+        "edge_face_connectivity",
+        "boundary_node_connectivity",
+        "volume_node_connectivity",
+        "volume_edge_connectivity",
+        "volume_face_connectivity",
+        "volume_volume_connectivity",
+    ]
+
+    def __init__(
+        self,
+        indices,
+        cf_role,
+        standard_name=None,
+        long_name=None,
+        var_name=None,
+        units=None,
+        attributes=None,
+        start_index=0,
+        src_dim=0,
+    ):
+        """
+        Constructs a single connectivity.
+
+        Args:
+
+        * indices (numpy.ndarray or numpy.ma.core.MaskedArray or dask.array.Array):
+            The index values describing a topological relationship. Constructed
+            of 2 dimensions - the list of locations, and within each location:
+            the indices of the 'target locations' it relates to.
+            Use a :class:`numpy.ma.core.MaskedArray` if :attr:`src_location`
+            lengths vary - mask unused index 'slots' within each
+            :attr:`src_location`. Use a :class:`dask.array.Array` to keep
+            indices 'lazy'.
+        * cf_role (str):
+            Denotes the topological relationship that this connectivity
+            describes. Made up of this array's locations, and the indexed
+            'target location' within each location.
+            Examples: ``face_node_connectivity``, ``edge_face_connectivity``.
```

Comment:
@trexfeathers You could change this line to simply say, 
```
See `Connectivity.UGRID_CF_ROLES`.
```

Reply:
6c9c76510b80fe3f9fae7a81ac1a449b0a577855